### PR TITLE
Return back Events Manager for CLI tasks

### PR DESCRIPTION
**Pull request by ruudboon**
Hello!

* Type: bug fix

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Added missing methods to Phalcon\Cli\Task. Placed tests in correct files and added more tests.

### CODE REVIEW

File: tests/_data/fixtures/Tasks/EchoTask.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+namespace Phalcon\Test\Fixtures\Tasks;
```
Comment (sergeyklay) [inline tests/_data/fixtures/Tasks/EchoTask.php]: Some PHP files has an empty line after `<?php`:
```php
<?php

namespace Phalcon\Test\Fixtures\Tasks;
```
but  some - not:
```php
<?php
namespace Phalcon\Test\Fixtures\Tasks;
```

Could you please use the same style.

File: tests/cli/Cli/Console/GetSetDefaultModuleCest.php [hunk 2]
```diff
@@ -32,9 +31,8 @@ class GetSetDefaultModuleCest
     public function cliConsoleSetGetDefaultModule(CliTester $I)
     {
         $I->wantToTest("Cli\Console - getDefaultModule()");
-        $I->wantToTest("Cli\Console - setDefaultModule()");
```
Comment (sergeyklay) [inline tests/cli/Cli/Console/GetSetDefaultModuleCest.php]: However this method calls getDefaultModule as well as setDefaultModule

**Review by sergeyklay — DISMISSED** [review]
There are minor corrections needs to be applied, but in general look good to me. Great work!

File: CHANGELOG-4.0.md [hunk 3]
```diff
@@ -35,6 +35,7 @@
 - Fixed `Phalcon\Firewall\Adapter\AdapterInterface::getRoleCallback` and `Phalcon\Firewall\Adapter\AbstractAdapter::setRoleCallback` to correctly accept and return a `Closure` [#14450](https://github.com/phalcon/cphalcon/issues/14450)
 - Fixed `Phalcon\Events\Event::__constructor` to correctly accept an `object` as the `source` parameter [#14449](https://github.com/phalcon/cphalcon/issues/14449)
 - Fixed `Phalcon\Cache::checkKey()` added `.` to key characters pattern [#14457](https://github.com/phalcon/cphalcon/pull/14457)
+- Fixed `Phalcon\Cli\Task::getEventsManager` and `Phalcon\Cli\Task::setEventsManager` added missing methods caused by `Phalcon\Di\Injectable` changes [#14445](https://github.com/phalcon/cphalcon/pull/14445)
```
[the remark this answers is not in the document]
Comment (ruudboon) [reply CHANGELOG-4.0.md]: Removed it

**Review by sergeyklay — APPROVED** [review]

**Pull request merged**